.eslintignore

```diff
@@ -40,6 +40,7 @@ target
 /x-pack/legacy/plugins/maps/public/vendor/**
 
 # package overrides
+/packages/elastic-safer-lodash-set/lodash/*
```

Comment:
@legrego This unfortunately also ignores the new license header check (see `.eslintrc.js`). Not sure if there's a way to ignore all checks except for some?

Reply:
> Not sure if there's a way to ignore all checks except for some?

Yeah, haven't heard of this either. Would it be feasible to add a `.eslintrc` file to `lodash` folder and specifically pick the rules we'd like to disable? It sounds like a tedious task, but this way we won't miss any new important rules that we may have in the future and would like to apply to forked code as well.

Reply:
@azasypkin Hmm I just realized that I could probably just add `/* eslint-disable */` after the header in each file in the `lodash` directory. That would effectively allow eslint to check the license is correct, but ignore everything else in the file. What do you think of that approach?

Reply:
I just pushed a commit with this implementation: https://github.com/elastic/kibana/pull/67452/commits/20d7b104fe73d64a2be96b8d7135caecf4900c90

Let me know what you think 🙂

Reply:
> /* eslint-disable */ after the header in each file in the lodash directory

Ah, I didn't expect that it just ignores everything _after_ this statement :slightly_smiling_face: Anyway I naively thought that editing one `.eslintrc` would be easier, but if approach in https://github.com/elastic/kibana/commit/20d7b104fe73d64a2be96b8d7135caecf4900c90 works for you then I'm fine too :+1: